### DISCUSSION TITLE
Allow me to hide myself from another persons For You feed

### DISCUSSION THREAD

**Discussion by Visne** [community/community#20074] [Other Feature Feedback, Questions, & Ideas]
I don't want every stupid thing I do to be visible in the feed of everyone. A basic privacy option to make my actions not show up in For You feeds would be nice.

**Comment by mtak** [comment]
In your profile [settings](https://github.com/settings/profile) you can hide activities:

![image](https://user-images.githubusercontent.com/1315327/177505707-ddaea03d-ada6-4235-b2ab-b81a9a855c6a.png)

**Reply by Commodore6416** [reply]
Thank you

**Reply by Visne** [reply]
I don't want to hide ALL my contributions

**Comment by Commodore6416** [comment]
Did I get there and fix up some mess
Fucking hope so
Thank again

**Comment by Commodore6416** [comment]
FYI for my own opsec I never keep an email, mone
